common/persistence/sql/sqlplugin/mysql/session/session.go

```diff
@@ -151,6 +152,10 @@ func buildDSN(
 
 func buildDSNAttrs(dbKind sqlplugin.DbKind, cfg *config.SQL) (map[string]string, error) {
 	attrs := make(map[string]string, len(dsnAttrOverrides)+len(cfg.ConnectAttributes)+1)
+	// Enable interpolation by default unless this is a mysql8 visibility store
```

Comment:
It would be neat to explain why? Something simple like "isn't compatible, see this issue" or sth would already help.

Reply:
This PR is built on top of https://github.com/temporalio/temporal/pull/5441 which explains the rationale

Reply:
Oh, I just mean in the code comment so it's easy to find (not everyone's going to git blame their way here).